SongLab/Components/Home/HomeViewModel.swift

```diff
@@ -7,13 +7,15 @@
 
 import Combine
 import Foundation
+import SwiftUI
 
 @MainActor
 class HomeViewModel: ObservableObject {
     
     //MARK: - API
     
     @Published var selectedSession: Session?
+    @Published var appTheme: String = "superglass"
```

Comment:
Depending on how the user will be able to select themes, the appTheme should probably be an environment variable.